src/ButtonAnchor.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default class ButtonAnchor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+  render() {
+    let { href, role, tabIndex, disabled, style } = this.props;
+
+    if (disabled) {
```

Comment:
The reason for it is that even tho the css will make it looked disabled it won't prevent focus, for disabled anchors. The `pointer-event: none` will be set on `a.btn.disabled` components by bootstrap css but not for anything else like MenuItems, where we'd also want to keep disabled ones from being navigatable by arrow keys and clicks.